xblock/scorable.py

```diff
@@ -0,0 +1,114 @@
+"""
+Scorable.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import namedtuple
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+Score = namedtuple('Score', ['raw_earned', 'raw_possible'])
+
+
+class ScorableXBlockMixin(object):
+    """
+    Mixin to handle functionality related to scoring.
+
+    Subclasses must define the following:
+
+        ScorableXBlockMixin.has_submitted_answer(self)
+        ScorableXBlockMixin.get_score(self)
+        ScorableXBlockMixin.set_score(self, score)
+        ScorableXBlockMixin.calculate_score(self, score)
+    """
+
+    has_score = True
```

Comment:
Nit: Can you add a docstring to explain the semantic of this class variable and why we need it?